.github/workflows/lint.yml

```
@@ -31,19 +31,16 @@ jobs:
         RUFF_VERSION=$(awk -F'[="]' '/\[project\.optional-dependencies\]/ {p=1} /ruff/ {if (p) print $4}' pyproject.toml)
         echo "RUFF_VERSION=$RUFF_VERSION" >> $GITHUB_ENV
 
-    - name: Install Ruff
-      run: >
-        curl --no-progress-meter --location --fail
-        --proto '=https' --tlsv1.2
-        --write-out "%{stderr}Downloaded: %{url}\n"
-        "https://astral.sh/ruff/$RUFF_VERSION/install.sh"
-        | sh
+    - name: Install Ruff ${{ env.RUFF_VERSION }}
+      uses: astral-sh/ruff-action@v3
+      with:
+        version: ${{ env.RUFF_VERSION }}
```

It seems by default this will run `check` (per https://github.com/astral-sh/ruff-action). I can't see an obvious way to disable this save running a different command, such as printing the version.
```suggestion
        args: --version
        version: ${{ env.RUFF_VERSION }}
```